README.md

```diff
@@ -34,63 +32,96 @@ message
 - timestamp
 - values[] (power, id, nameHash, value)
 
-## Chain Contract Addresses
-
-1. Deploy ResultManager contract on source chain using `deployResultManager.js` script.
-   > **_NOTE:_** Please verify SIGNER_ADDRESS and DESTINATION_CHAIN before deploying in the script.
-2. Deploy Forwarder, TransparentProxy and Staking contracts using `deployForwarder.js` script.
-   > **_NOTE:_** Verify SIGNER_ADDRESS and DESTINATION_CHAIN in script before deploying.
-
-## Contract Address
-
-###  Calypso Testnet
-
-| Contract                       | Address                                    |
-| ------------------------------ | ------------------------------------------ | 
-| ResultManager         | 0xFEb70E7Ad36b425b698184B3d5b1F856c8132a42 | 
-| Forwarder  | 0xa26232204879C4370806cA56F821b9fe97eAeF5B |
-| TransparentForwarder                       | 0xbF5c5AD799b2245BA36562BebfcbAbc5D508Eb84 |
-| Staking           | 0x03bD73afB6d3C5B86578010C099E5474aF1aABeF | 
-
-
-###  Europa Testnet
-
-| Contract                       | Address                                    |
-| ------------------------------ | ------------------------------------------ | 
-| ResultManager         | 0xFEb70E7Ad36b425b698184B3d5b1F856c8132a42 | 
-| Forwarder  | 0xa26232204879C4370806cA56F821b9fe97eAeF5B |
-| TransparentForwarder                       | 0xbF5c5AD799b2245BA36562BebfcbAbc5D508Eb84 |
-| Staking           | 0x03bD73afB6d3C5B86578010C099E5474aF1aABeF | 
+## Deployment
 
+Deploy centralised bridge on destination chain using `deployBridge.js` script.
```

Comment:
@rajkharvar not sure if should use the term `centralised` here cc @SkandaBhat

Reply:
Resolved in [afdebf5](https://github.com/razor-network/bridge-proxy/pull/49/commits/afdebf50c86bf101fe23162b3fa319e2b7ac84dd)